blocks/library/table/table-block.js

```diff
@@ -89,6 +89,7 @@ export default class TableBlock extends wp.element.Component {
 						<li>
 							<DropdownMenu
 								icon="editor-table"
+								label={ wp.i18n.__( 'Edit Table' ) }
```

Comment:
Yep I know, I didn't want to change also all the other ones, because outside of the scope of this PR. But since we're here, I can do that if you agree.